crates/forge/tests/e2e/common/runner.rs

```diff
@@ -32,25 +33,17 @@ pub(crate) fn setup_package_with_file_patterns(
         .replace('\\', "/");
 
     let manifest_path = temp.child("Scarb.toml");
-    manifest_path
-        .write_str(&formatdoc!(
-            r#"
-                [package]
-                name = "{}"
-                version = "0.1.0"
-
-                [[target.starknet-contract]]
-                sierra = true
-                casm = true
 
-                [dependencies]
-                starknet = "2.2.0"
-                snforge_std = {{ path = "{}" }}
-                "#,
-            package_name,
-            snforge_std_path
-        ))
+    let mut scarb_toml = fs::read_to_string(&manifest_path)
```

Comment:
[[Nitpick]] I would remove it from this PR as it is completely unrelated and makes commit history messy, though knowing the reasons why its here I assume it would be a bit annoying so marking it as nit.